server/server.go

```diff
@@ -181,6 +182,15 @@ func (s *Server) Serve() error {
 	select {}
 }
 
+func (s *Server) Start() error {
+	if err := s.exportServices(); err != nil {
+		return err
+	}
+	metadata.ExportMetadataService()
+	registry_exposed.RegisterServiceInstance(s.cfg.Application.Name, s.cfg.Application.Tag, s.cfg.Application.MetadataType)
```

Comment:
This part is used for what?

Reply:
I have the same question, we already have `server.Serve()`. And also `dubbo.Start`, `dubbo.LoadProvider`,`dubbo.LoadConsumer` are not necesary.

Reply:
Non-blocking start, same as the original

Reply:
I need a func to start without blocking

Reply:
How about you we make the new method private and delegate the main logic of `Serve` to this method by calling `ins.start()`?

Reply:
```config.Load``` is non-blocking. We could put ```server.Serve()``` in a goroutine to make ```dubbo.Load()``` non-blocking. Then there would be enough space for us to consider whether introducing a non-blocking method ```server.Start()``` is necessary.